raiden/ui/web/src/app/components/channel-table/channel-table.component.ts

```diff
@@ -1,23 +1,22 @@
-import { Observable, BehaviorSubject } from 'rxjs';
-import { scan, tap, switchMap, map, finalize } from 'rxjs/operators';
-import { Component, OnInit } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { MenuItem } from 'primeng/primeng';
-
-import { RaidenConfig } from '../../services/raiden.config';
-import { RaidenService } from '../../services/raiden.service';
-import { SharedService } from '../../services/shared.service';
+import { BehaviorSubject } from 'rxjs/internal/BehaviorSubject';
```

Comment:
I think you can and should import these objects directly from the `rxjs` path, no need to mess with `/internal/` paths

Reply:
Actually there reason for this change is due to the lint rules. If you check the lint rules in [tslint.json](https://github.com/raiden-network/raiden/blob/master/raiden/ui/web/tslint.json#L19) it has the import of `rxjs` and `rxjs/Rx` blacklisted. 

If you go and run lint checks with `ng lint` you will see it listing a lot of `This import is blacklisted, import a submodule instead`.

Apparently this is done to reduce the final bundle size according to this [comment](https://github.com/angular/angular/issues/20349#issuecomment-343638051).

In our case I believe bundle doesn't matter, as long as the WebUI runs on the local machine, but since this is the default accepted behavior, I think we should follow it unless there is a valid reason not to do so.

Reply:
Oh, right.. looks like there's some discrepancy between documentation and linting then. No problem anyway, thank you for pointing this out.